use-cases.md

```diff
@@ -53,11 +53,7 @@ When a server wants to trace execution of a request, it generally needs to go th
 Let's assume that we have an HTTP server, and the Span is propagated from the client via HTTP headers, accessible via `request.headers`:
 
 ```python
-    span = tracer.join_trace_from_text(
-        operation_name=operation,
-        context_snapshot=request.headers,
-        trace_attributes=request.headers
-    )
+    span = tracer.get_span_extractor(request.class).extract(request)
```

Comment:
> only by delegating to some other "known" injectors provided by the tracer, as ultimately the app has no idea how to encode the distributed context into binary or kv-map.

there's further journey to be had here, but this is going into more the internal API side of things where we have more freedom to make mistakes.